### PR TITLE
mute and push config api

### PR DESCRIPTION
**Pull request by jimxj**
@priteshshah1983 @lstanii-magnet @pekboy  Please review

### CODE REVIEW

**Comment by priteshshah1983** [comment]
Looks good

**Comment by jimxj** [comment]
@priteshshah1983  Please review again.  Ignore the changes merged from poll implementation.

File: android/src/main/java/com/magnet/mmx/client/api/MMXChannel.java [hunk 1]
```diff
@@ -463,6 +482,22 @@ public Date getCreationDate() {
...
+   */
+  public boolean isMuted() {
+    return mIsMuted;
+  }
+
+  /**
+   * The name of the push configuration (match what on server side)
+   * @return
+   */
+  public String getPushConfigName() {
```
Comment (priteshshah1983) [inline android/src/main/java/com/magnet/mmx/client/api/MMXChannel.java]: We are not going to expose this getter for now.

**Comment by priteshshah1983** [comment]
If I'm ignoring what I'm supposed to ignore correctly, looks good to me, other than my 1 comment that I posted about the getter.

**Pull request merged**